### PR TITLE
Delete recurring weekly and update User Guide

### PR DESCRIPTION
**Pull request by SukiTsang**

### CODE REVIEW

**Review by Skaty — APPROVED** [review]
:+1:

File: docs/UserGuide.md [hunk 1]
```diff
@@ -23,7 +23,7 @@
 <!-- @@author A0153467Y -->
 ## Introduction
 
-Nowadays, everyone has so many tasks to do. Have you ever missed your deadlines or forget your schedule? MESS is here to help you! MESS is a to-do list application which reminds you of your tasks or events by showing your list of tasks on the application. No need to afraid of forgetting your tasks anymore! You can easily use MESS by typing in one line of command. MESS can be used offline and even in your office computer. It helps you manage and organise your tasks and so as your time. 
+Nowadays, everyone has so many tasks to do. Have you ever missed your deadlines or forget your schedule? MESS is here to help you! MESS is a to-do list application which reminds you of your tasks or events by showing your list of tasks on the application. No need to afraid of forgetting your tasks anymore! You can easily use MESS by typing only one line of command. MESS can be used offline and even in your office computer. It helps you to manage and organise your tasks and so as your time.
```
Comment (advaypal) [inline docs/UserGuide.md]: Change to this:

Nowadays, everyone has so many tasks to do. Have you ever missed your deadlines or forgotten your schedule? If so, MESS is here to help you! MESS is a to-do list application which reminds you of your tasks or events by showing your list of tasks on the application. No need to be afraid of forgetting your tasks anymore! You can easily use MESS through your keyboard by typing only simple and short commands. MESS can be used offline and even on your office computer. It helps you manage and organise both your tasks and your time.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -69,23 +69,24 @@ Want to use MESS immediately? Let's get started!
  
 #### Adding a task or event: `add`
 Adds a task to the to-do list<br>
-Format: `add TASK_NAME [starts START_DATETIME ends CLOSE_DATETIME tag TAG recurs NUMBER_OF_WEEKLY_RECURRING_TASK]`
+Format: `add TASK_NAME [starts START_DATETIME ends CLOSE_DATETIME tag TAG]`
 
 > Date format of START_DATE and CLOSE_DATE includes words like today, tomorrow, 3 days from now, day after tomorrow, noon, 12pm, 6am
 
+> Use ISO date-time format for typing exact date. e.g. 2016/9/1 represents 1st September 2016
```
Comment (advaypal) [inline docs/UserGuide.md]: Change to

Use ISO date-time format if you wish to type exact date. e.g. 2016/9/1 represents 1st September 2016

**Comment by advaypal** [comment]
@SukiTsang Please make the necessary changes

**Review by advaypal — CHANGES_REQUESTED** [review]
@SukiTsang Please make the necessary changes

File: docs/UserGuide.md (hunk 1 above)
Comment (SukiTsang) [reply docs/UserGuide.md]: ok

**Comment by SukiTsang** [comment]
@advaypal please review again

**Review by advaypal — APPROVED** [review]

**Pull request merged**